icons/kotlin.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><linearGradient id="a" x1="1.725" x2="22.185" y1="22.67" y2="1.982" gradientTransform="translate(1.306 1.129) scale(.89324)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#0296d8"/><stop offset="1" stop-color="#8371d9"/></linearGradient><linearGradient id="b" x1="1.869" x2="22.798" y1="22.382" y2="3.377" gradientTransform="translate(1.323 1.129) scale(.89324)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#cb55c0"/><stop offset="1" stop-color="#f28e0e"/></linearGradient></defs><path fill="url(#a)" d="M2.975 2.976v18.048h18.05v-.03l-4.478-4.511-4.48-4.515 4.48-4.515 4.443-4.477z"/><path fill="url(#b)" d="m12.223 2.976-9.23 9.23v8.818h.083l9.032-9.032-.024-.024 4.48-4.515 4.443-4.477h-8.784z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><defs><linearGradient id="a" x1="1.725" x2="22.185" y1="22.67" y2="1.982" gradientTransform="translate(1.306 1.129) scale(.89324)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#7f52ff"/><stop offset="0.5" stop-color="#c711e1"/><stop offset="1" stop-color="#e44857"/></linearGradient></defs><path fill="url(#a)" d="M2.975 2.976v18.048h18.05v-.03l-4.478-4.511-4.48-4.515 4.48-4.515 4.443-4.477z"/></svg>
```

Comment:
Check the [Color Check report](https://github.com/material-extensions/vscode-material-icon-theme/actions/runs/11653887169/job/32456453538?pr=2666)

It suggest replacement colors from the Material Palette. You can also use Pkief's [Material Color Convertor](https://pkief.github.io/material-color-converter/) tool

Reply:
> Check the [Color Check report](https://github.com/material-extensions/vscode-material-icon-theme/actions/runs/11653887169/job/32456453538?pr=2666)
> 
> It suggest replacement colors from the Material Palette. You can also use Pkief's [Material Color Convertor](https://pkief.github.io/material-color-converter/) tool

If it is a branded icon of the brand, then you can leave its original colors, there were such cases, for example, with the **Vue** icon, its colors remained original

Reply:
I updated the colors, but would you prefer the original, or should I keep the Material Palette version?